src/languages/en.js

```diff
@@ -542,6 +542,7 @@ export default {
             firstNameLength: 'First name shouldn\'t be longer than 50 characters',
             lastNameLength: 'Last name shouldn\'t be longer than 50 characters',
             characterLimit: ({limit}) => `Exceeds the max length of ${limit} characters`,
+            hasCommas: 'Please remove the comma from the name field.',
```

Comment:
I think we could add the semicolon here, but also we/I could update the semicolon in App too once this is done to separate concerns

Reply:
I think I might as well do it here, it's just updating the regex and the copy